_layouts/main.html

```diff
@@ -1,22 +1,21 @@
 <!DOCTYPE html>
 <html lang="{{ page.lang | default: site.lang | default: 'en-US' }}">
   <head>
-    {% include meta.html %}
-    {% include styles.html %}
-    {% include google_analytics.html %}
-    {% include google_recaptcha.html %}
+    {% include meta.html %} {% include styles.html %} {% include
+    google_analytics.html %} {% include google_recaptcha.html %}
```

Comment:
ditto, can we bring back the old formatting with the newlines?

Reply:
Done. Will add rules for new lines when I add ESLint and Stylelint. Keep an 👁️  out for a future PR.